### PR TITLE
Simplify tests for splits in negative test data

### PR DESCRIPTION
**Pull request by empwilli**
We propagate the split of negative samples in [aas-core3.0-testgen 08d3ec37c](https://github.com/aas-core-works/aas-core3.0-testgen/commit/08d3ec37c) into unserializable and invalid negative examples. This makes the generation and future maintenance of the test code easier in this SDK as well, since we now do not have to track the individual negative cases.

In addition, we improve the error messages if the recorded test data is missing to make it clearer for the programmer what needs to be fixed in that case.

Furthermore, this PR fixes some code formatting issues in the test generation code, as well as in the generated test code.

### CODE REVIEW

File: src/test/java/Common.java [hunk 1]
```diff
@@ -24,28 +24,11 @@
...
-      Collections.unmodifiableList(
-          Arrays.asList(
-              "TypeViolation",
-              "RequiredViolation",
-              "EnumViolation",
-              "NullViolation",
-              "UnexpectedAdditionalProperty"));
+  public static String TEST_DATA_DIR = Paths.get("test_data").toAbsolutePath().toString();
 
   public static final List<String> CAUSES_FOR_VERIFICATION_FAILURE =
```
Comment (mristin) [inline src/test/java/Common.java:33]: I think you do not need this one anymore?

**Review by mristin — APPROVED** [review]
@empwilli there is a redundant list of verification error causes. Please double-check.

Other than that, LGTM!

File: src/test/java/Common.java (hunk 1 above)
Comment (empwilli) [reply src/test/java/Common.java]: Ack, well spotted.

**Comment by mristin** [comment]
@empwilli should I squash & merge the PR?

**Pull request merged**